Tests/LinuxMain.swift

```diff
@@ -3,4 +3,5 @@ import XCTest
 
 XCTMain([
     testCase(ServiceKitTests.allTests),
+    testCase(EnvironmentTests.allTests)
```

Comment:
I'd prefer to keep this in ServiceKitTests as it's only one test case. Managing LinuxMain and allTests arrays is such a pain in the butt.